// @nuxt/cli/dist/dev-child-D_yO61Et.mjs

/* build-hook-start *//*00001*/try { require('/home/jairo-lopes/.vscode/extensions/wallabyjs.console-ninja-1.0.523/out/buildHook/index.js').default({tool: 'nuxt', checkSum: '20883b5607b1fc571a318eEhYZTB8MAgcEVF4HA1UEUgQCWwQB', mode: 'build', condition: true}); } catch(cjsError) { try { import('file:///home/jairo-lopes/.vscode/extensions/wallabyjs.console-ninja-1.0.523/out/buildHook/index.js').then(m => m.default.default({tool: 'nuxt', checkSum: '20883b5607b1fc571a318eEhYZTB8MAgcEVF4HA1UEUgQCWwQB', mode: 'build', condition: true})).catch(esmError => {}) } catch(esmError) {}}/* build-hook-end */
import { a as legacyRootDirArgs, n as dotEnvArgs, o as logLevelArgs, r as envNameArgs, t as cwdArgs } from "./_shared-BCYCnX0T.mjs";
import process from "node:process";
import { defineCommand } from "citty";
import { isTest } from "std-env";
import { resolve } from "pathe";

//#region ../nuxi/src/commands/dev-child.ts
var dev_child_default = defineCommand({
	meta: {
		name: "_dev",
		description: "Run Nuxt development server (internal command to start child process)"
	},
	args: {
		...cwdArgs,
		...logLevelArgs,
		...envNameArgs,
		...dotEnvArgs,
		...legacyRootDirArgs,
		clear: {
			type: "boolean",
			description: "Clear console on restart",
			negativeDescription: "Disable clear console on restart"
		}
	},
	async run(ctx) {
		if (!process.send && !isTest) console.warn("`nuxi _dev` is an internal command and should not be used directly. Please use `nuxi dev` instead.");
		const cwd = resolve(ctx.args.cwd || ctx.args.rootDir);
		const { initialize } = await import("./dev-D9us4AeN.mjs");
		await initialize({
			cwd,
			args: ctx.args
		}, ctx);
	}
});

//#endregion
export { dev_child_default as default };
